### DISCUSSION TITLE
Zombie Mode

### DISCUSSION THREAD

**Discussion by Icenova20** [make-all/tuya-local#2487] [Q&A]
Has anyone found a way to reliable configure a network to minimize Zombie mode? (the device becomes unresponsive if it cant reach tuya servers)

Specifically i have a Peephole camera and if it can't reach servers it never starts it's RTSP stream which makes it very useless. I've read that disabling DNS access is a possible path forward, but i'm not sure how one would do that on a device that Requires DHCP because it is a wifi only device without a way to set it up outside of the tuya app. I would love to answer any questions to try and see what i'm missing here, but i dont see a way get this device to work offline unless somehow emulating the tuya servers? not sure where to even begin there.

**Comment by make-all** [comment]
To do advanced things like this on your network, you will need to turn off the DHCP in your router and run a separate DHCP and/or DNS server that can be configured down to individual devices.
